tests/lib.rs

```diff
@@ -85,10 +85,14 @@ impl Arbitrary for Ma {
 #[derive(PartialEq, Eq, Clone, Debug)]
 struct Proto(Protocol<'static>);
 
+impl Proto {
+    const IMPL_VARIANT_COUNT: u8 = 32;
```

Comment:
I was actually thinking of directly on the enum as `Protocol::VARIANT_COUNT`.

I am sure there are also proc-macros for this that we could enable only during testing if this nightly stuff is too fancy 😁

Reply:
but `Protocol` lives in the crate, the `VARIANT_COUNT` refers to `Proto`'s arbitrary implementation count. `Protocol`'s count is what we get with `mem::variant_count` you mean implementing it on `Protocol` in the tests?

Reply:
Right, that would unnecessarily increase the public API. Leave it as is! :)